src/main/java/seedu/address/logic/commands/SearchCommand.java

```diff
@@ -26,13 +26,13 @@ public class SearchCommand extends Command {
             + ": Searches all persons whose names or tags contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "At least one of the following parameters must be provided.\n"
-            + "Parameters: " + "PREFIX/ KEYWORD [MORE_PREFIX/ KEYWORD...] \n"
-            + "You can search by name, tag, or both."
-            + " If both are provided, only persons matching both criteria will be shown.\n"
-            + "Example 1: " + COMMAND_WORD + " n/ alice\n"
-            + "Example 2: " + COMMAND_WORD + " t/ friend\n"
-            + "Example 3: " + COMMAND_WORD + " n/ alice t/ friend\n"
-            + "Example 4: " + COMMAND_WORD + " t/ friend n/ alice";
+            + "Parameters: " + "PREFIX/KEYWORD [MORE_PREFIX/KEYWORD...] \n"
+            + "You can search by name, tag, role, phone number or by group."
+            + " If multiple prefixes are provided, only persons who match all prefixes will be shown.\n"
+            + "Example 1: " + COMMAND_WORD + " n/alice\n" //show single search
+            + "Example 2: " + COMMAND_WORD + " g/blood drive\n"
+            + "Example 3: " + COMMAND_WORD + " n/alice t/friend p/91234567 r/volunteer\n" // show that can stack search
+            + "Example 4: " + COMMAND_WORD + " r/volunteer t/friend p/91234567 n/alice"; // show order does not matter
 
```

Comment:
A little strange for UG to have comments?

Reply:
I think there is no need for so many examples as well

Reply:
oh the comments are not visible to the user, but i see what you mean ill just remove it. regarding the issue of too many examples, do you think its fine to omit example 2 and 4? (since example 4 where the order does not matter is stated in the UG)

Reply:
Yes sounds good